modules/apps/29-fee/ibc_module_test.go

```diff
@@ -719,27 +717,18 @@ func (suite *FeeTestSuite) TestOnTimeoutPacket() {
 	for _, tc := range testCases {
 		tc := tc
 		suite.Run(tc.name, func() {
-			suite.SetupTest()
-
-			// open incentivized channel
+			suite.SetupMockTest()
 			suite.coordinator.Setup(suite.path)
-
-			// set up coin & create ics20 packet
-			coin := ibctesting.TestCoin
-			packet := suite.CreateICS20Packet(coin)
-
-			// setup for ics20: fund chain A's escrow path so that tokens can be unescrowed upon timeout
-			escrow := transfertypes.GetEscrowAddress(suite.path.EndpointA.ChannelConfig.PortID, suite.path.EndpointA.ChannelID)
-			suite.Require().NoError(simapp.FundAccount(suite.chainA.GetSimApp(), suite.chainA.GetContext(), escrow, sdk.NewCoins(coin)))
```

Comment:
no longer needed. It was only there so the transfer OnTimeoutPacket callback wouldn't error